tests/functional/raw-node/test/GCP/bucket/get.js

```diff
@@ -0,0 +1,176 @@
+const assert = require('assert');
+const async = require('async');
+const { GCP } = require('../../../../../../lib/data/external/GCP');
+const { makeGcpRequest } = require('../../../utils/makeRequest');
+const { gcpRequestRetry } = require('../../../utils/gcpUtils');
+const { getRealAwsConfig } =
+    require('../../../../aws-node-sdk/test/support/awsConfig');
+const constants = require('../../../../../../constants');
+
+const credentialOne = 'gcpbackend';
+
+describe('GCP: GET Bucket', function testSuite() {
+    this.timeout(180000);
+    const config = getRealAwsConfig(credentialOne);
+    const gcpClient = new GCP(config);
+
+    describe('without existing bucket', () => {
+        beforeEach(function beforeFn(done) {
+            this.currentTest.bucketName = `somebucket-${Date.now()}`;
+            return done();
+        });
+
+        it('should return 404 and NoSuchBucket', function testFn(done) {
+            gcpClient.getBucket({
+                Bucket: this.test.bucketName,
+            }, err => {
+                assert(err);
+                assert.strictEqual(err.statusCode, 404);
+                assert.strictEqual(err.code, 'NoSuchBucket');
+                return done();
+            });
+        });
+    });
+
+    describe('with existing bucket', () => {
+        let numberObjects;
+        beforeEach(function beforeFn(done) {
+            this.currentTest.bucketName = `somebucket-${Date.now()}`;
+            this.currentTest.createdObjects = Array.from(
+                Array(numberObjects).keys()).map(i => `someObject-${i}`);
+            process.stdout
```

Comment:
are u using these to debug?

Reply:
Initially, yes. It also served as an indicator that something is happening, so I left it in the code.